src/components/renderer/renderer.tsx

```diff
@@ -139,7 +140,24 @@ class Editor extends React.PureComponent<Props, State> {
     if (params[params.length - 1] === 'view') {
       this.setState({ fullscreen: true });
     }
+
+    window.addEventListener(
+      'popstate',
```

Comment:
What is this event and why can't we move this code where this event is triggered?

Reply:
This event is triggered on back button press and that is handled internally by browser

Reply:
But doesn't react route listen to this already? Can't we use that?

Reply:
I added it. Please update the types.